npm link is broken

Symbol upload broke on our build machine with the following error:

```

```

I checked the password, and it appeared to be correct. When attempting to debug locally I noticed several package issues when attempting to consume the package via `npm link`:

```
bobby@MrPresidentsMBP bugsplat-web-app % npm run symbols

> bugsplat-web-app@2.2.21 symbols
> node ./symbols.js

node:internal/modules/cjs/loader:944
  throw err;
  ^

Error: Cannot find module 'form-data'
Require stack:
- /Users/bobby/Desktop/bugsplat/symbol-upload/dist/lib/bugsplat-api-client.js
- /Users/bobby/Desktop/bugsplat/symbol-upload/dist/index.js
- /Users/bobby/Desktop/bugsplat/bugsplat-web-app/symbols.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:941:15)
    at Function.Module._load (node:internal/modules/cjs/loader:774:27)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/bobby/Desktop/bugsplat/symbol-upload/dist/lib/bugsplat-api-client.js:8:37)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/bobby/Desktop/bugsplat/symbol-upload/dist/lib/bugsplat-api-client.js',
    '/Users/bobby/Desktop/bugsplat/symbol-upload/dist/index.js',
    '/Users/bobby/Desktop/bugsplat/bugsplat-web-app/symbols.js'
  ]
}
```

<!-- Edit the body of your new issue then click the ✓ "Create Issue" button in the top right of the editor. The first line will be the issue title. Assignees and Labels follow after a blank line. Leave an empty line before beginning the body of the issue. -->

--- a/package.json
+++ b/package.json
@@ -2,13 +2,16 @@
   "name": "@bugsplat/symbol-upload",
   "version": "2.1.0",
   "description": "Cross platform symbol upload tool",
-  "main": "index.ts",
+  "main": "./dist/index.js",
   "bin": {
-    "symbol-upload": "bin/index.js"
+    "symbol-upload": "./dist/bin/index.js"
   },
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "rm -rf ./dist && tsc &&  cp ./{package.json,README.md,LICENSE} ./dist",
+    "build": "rm -rf ./dist && tsc",
     "release": "cd ./dist && npm publish --access public"
   },
   "repository": {
@@ -30,13 +33,14 @@
   },
   "homepage": "https://github.com/BugSplat-Git/symbol-upload#readme",
   "devDependencies": {
-    "@types/node": "^14.14.37",
-    "@types/glob": "^7.1.3"
+    "@types/glob": "^7.1.3",
+    "@types/node": "^14.14.37"
   },
   "dependencies": {
     "form-data": "^4.0.0",
     "glob": "^7.1.6",
     "node-fetch": "^2.6.1",
-    "rxjs": "^6.6.6"
+    "rxjs": "^6.6.6",
+    "typescript": "^4.3.2"
   }
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -9,6 +9,10 @@
     "noImplicitAny": false,
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true
-  }
+    "forceConsistentCasingInFileNames": true,
+  },
+  "files": [
+    "./index.ts",
+    "./bin/index.ts"
+  ]
 }

